prime-angular-frontend/src/app/lib/constants.ts

```diff
@@ -22,3 +22,9 @@ export const DISPLAY_ID_OFFSET = 1000;
  * Paper enrollee GPID filter.
  */
 export const PAPER_ENROLLEE_GPID_PREFIX = 'NOBCSC';
+
+/**
+ * @description
+ * ngBusy submission message
+ */
+export const NG_BUSY_SUBMISSION_MESSAGE = 'Wait patiently, the submission can take up to a minute';
```

Comment:
Maybe nitpicky but do we want to tie constants or variable names to an implementation?  e.g. this could just be `BUSY_SUBMISSION_MESSAGE`

Reply:
I was also thinking if whether we should make it a constant in the **constants.ts** file or in the module itself so that we can change what the text is based on our needs in the future.

Reply:
I like how it is now, in `src\app\lib\modules\ngx-busy\busy.config.ts`